### ISSUE TITLE
Nicehash Miner 2 Won't Launch

### ISSUE THREAD

**Issue by 3dreamsim** [nicehash/NiceHashMiner-Archived#1331]
Im a noob when it comes to this, but here is my issue. I click to run the miner, it loads up to put the numbers in and click confirm. It loads up till it says querying opencl devices then it will sit for like 1 min then just close? Anyone have this happen before? Am I missing something?

**Comment by dsedov** [comment]
Yes. I have exactly the same problem. What helped me is deleting config files

**Comment by KanekiK3n** [comment]
I have the same problem but deleting the config files didn't help me though.

**Comment by kunambi** [comment]
This happens every time I start Nicehash 2.0.15:

![image](https://user-images.githubusercontent.com/739112/33667353-8f01a204-dad7-11e7-9b00-42f5587fe992.png)

I think this is because my Nvidia GTX 1070 aren't supporting OpenCL. 

![image](https://user-images.githubusercontent.com/739112/33667392-abe44b60-dad7-11e7-8e80-a1aa9d402f82.png)

Is there a way to turn off the OpenCL-check, or will the entire application become useless? Trying to start Nicehash miner 1.7.5.2 seems to work better—I'm stuck on "SMA information" but I think that's because of the current outtage.

TYIA

**Comment by utmath** [comment]
@dsedov Would you tell me where the config file exist?
